src/service/game.go

```diff
@@ -14,31 +14,79 @@ import (
 func NewGameService(r repository.Repository, rand RandomService, max int) GameService {
 	v := validator.New()
 	return &service{
-		Repo: r,
-		Validate: v,
+		Repo:         r,
+		Validate:     v,
 		MaximumCards: max,
-		Rand: rand,
+		Rand:         rand,
 	}
 }
 
-// GameService is interface with methods to interact with redis db	
+// GameService is interface with methods to interact with redis db
 type GameService interface {
-    NewGame(input *TeamInput) (string, error)
+	NewGame(input *TeamInput) (string, error)
 	GetGame(gameID string) (game *Game, err error)
 	UpdateGame(gameID string, input *GameInput) (*Game, error)
 
 	SaveCard(gameID string, card *CardInput) (string, error)
 	GetRandomCard(gameID string) (card *Card, err error)
+<<<<<<< HEAD
+	MakeRandomCards(gameID string, numCards int) (game *Game, err error)
+	MarkCardUsed(gameID, cardID string) error
+=======
 	MarkCardUsed(gameID, cardID string) (*Game, error)
+>>>>>>> f9d1865babbc83d1f03dc600a9a800e9cb036b61
```

Comment:
I took a look at the merge conflicts and I think it might be caused by the change I made to the `MarkCardUsed` service method last weekend - in your branch it looks like this method only returns an `error` but in the staging branch it returns a `*Game` and `error`. I think if you rebase on staging locally you'll be able to see the differences in VS Code - after you edit the file to pick which change you want to keep and remove the "diff" lines you'll just want to save the change and force push the updated branch. (It looks like right now the "diff" might have been saved here by accident)
Should be something like:
```bash
$ git checkout staging
$ git pull origin staging
$ git checkout RandomGame
$ git rebase staging 
$ git status                    # You'll see all the files which have conflicts bw staging and RandomGame. Open the files and remove everything except the changes you want to stick with and save files
$ git add < FILES WITH RESOLVED CONFLICT >
$ git commit -m 'Resolved merge conflict'
$ git rebase --continue
$ git status                    # Should see everything is up-to-date
$ git log                       # Should see my commit to Merge pull request #34 (https://github.com/TifMoe/go-fishbowl/commit/f9d1865babbc83d1f03dc600a9a800e9cb036b61)
$ git push -f origin RandomGame # Force push because you've altered history of your branch
```
Happy to walk through it on a call if that's helpful!!